app/views/layouts/basic_template.html.erb

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en-US">
+
+  <head>
+    <meta charset="UTF-8">
+    <title><%= yield :template %> | Madeline System</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <%= yield :head %>
+    <%= stylesheet_link_tag "dashboard", media: "all" %>
+  </head>
+
+  <body>
+    <%= render "layouts/main_content" %>
+  </body>
+
+</html>
```

Comment:
Actually, it seems like these `basic_layoutare almost exact duplicates of application